arangod/Scheduler/SupervisedScheduler.h

```diff
@@ -81,13 +81,9 @@ class SupervisedScheduler : public Scheduler {
   // in a container class and store pointers. -- Maybe there is a better way?
   boost::lockfree::queue<WorkItem*> _queue[3];
 
-  char _padding1[64];
```

Comment:
those were probably here intentionally, even if not used.
I guess the purpose of these variables was to prevent false sharing.

Reply:
so leave as is?

Reply:
Instead of defining members for the padding you could specify the alignment of the members: 
```cpp
  alignas(64) std::atomic<uint64_t> _jobsSubmitted;
  alignas(64) std::atomic<uint64_t> _jobsDequeued;
  alignas(64) std::atomic<uint64_t> _jobsDone;

  // ...
  alignas(64) std::atomic<uint64_t> _wakeupQueueLength;
```
This not only removes the unused members, but also reduces the padding between the members to the required minimum (i.e., in these cases 56 bytes instead of 64 bytes).